### PR TITLE
WIP - minimal+base+sdk+proxy_SCC-postreg for SLE15

### PR DESCRIPTION
**Pull request by yxususe**
Solution for https://progress.opensuse.org/issues/19566.

### CODE REVIEW

**Comment by yxususe** [comment]
Testruns: 
http://f146.suse.de/tests/1185
http://f146.suse.de/tests/1186

File: tests/console/proxyscc_postreg.pm [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2014-2017 SUSE LLC
```
Comment (rwx788) [inline tests/console/proxyscc_postreg.pm]: Use only 2017 here

File: tests/console/proxyscc_postreg.pm [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Do the registration against SCC after installation
```
Comment (rwx788) [inline tests/console/proxyscc_postreg.pm]: Add details, that registration is done with SUSEConnect tool and not with yast scc.

**Comment by yxususe** [comment]
Basically I agree with Rodion, that one either register modules with SuSEConnect (when sle version>=15), or with yast scc. However Oliver has different opinions.

**Comment by rwx788** [comment]
@yxususe SuSEConnect tool can be used with SLE12 as well, but modules are obviously not there. I'm not sure if @okurz has different opinion here, we just need separate test suite. Let's when till Monday.

**Comment by okurz** [comment]
yes, please wait until Monday and let's rediscuss then.

File: tests/console/proxyscc_postreg.pm [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+sub run {
+    return if get_var("HDD_SCC_REGISTERED");
+    my $version  = get_required_var("VERSION");
+    my $arch     = get_required_var("ARCH");
+    my $reg_code = get_required_var("SCC_REGCODE");
+    my $scc_url  = get_var("SCC_URL");
+
+    select_console 'root-console';
+    assert_script_run "SUSEConnect --url $scc_url" if $scc_url;
```
Comment (rwx788) [inline tests/console/proxyscc_postreg.pm]: I've tested this in latest builds, not sure how it worked before, but if we redefined $scc_url, we should provide reg code within same command. So, it should be "SUSEConnect --url $scc_url -r $reg_code". Please, change this accordingly, but still having a fallback to registration against official SCC when SCC_URL is not defined.

**Comment by yxususe** [comment]
Somehow I did some wrong operation and I will open a new PR for it.

**Pull request closed without merging**